meson_options.txt

```diff
@@ -17,6 +17,6 @@ option('enable-python', type: 'boolean', value: true)
 option('enable-tizen', type: 'boolean', value: false)
 option('enable-element-restriction', type: 'boolean', value: false) # true to restrict gst-elements in api
 option('restricted-elements', type: 'string', value: '')
-option('enable-nnfw', type: 'boolean', value: false)
-option('enable-nnfw-runtime', type: 'boolean', value: false)
+option('enable-nnfw-tflite', type: 'boolean', value: false) # true to enable nnfw backend with tflite filter
```

Comment:
Based on the above assumption, it'd be # true to enable tensorflow-lite to delegate nnapi interpretation to its backend; i.e., nnfw backend in Tizen for tensorflow-lite.

Reply:
updated.